src/flatc.cpp

```diff
@@ -157,6 +157,7 @@ std::string FlatCompiler::GetUsageString(const char *program_name) const {
     "  --include-prefix       Prefix this path to any generated include statements.\n"
     "    PATH\n"
     "  --keep-prefix          Keep original prefix of schema include statement.\n"
+    "  --keep-namespaces      Keep full namespace paths of imported types in Rust.\n"
```

Comment:
Can we make these more explicitly for Rust only? It seems like JS and Go use flags with their name in them.

Reply:
I thought about it, but figured a generic flag would be potentially reusable in other languages. Otherwise we would need to keep a family of language-specific flags.

Reply:
if anything, we have some other flags that apply to multiple langs, so we could choose to merge some flags which essentially do the same. `--do-something   Do the something (Go, JS, C#)`.